bin/informational/patron_information

```diff
@@ -16,7 +16,7 @@ from api.authenticator import LibraryAuthenticator, PatronData
 class PatronInformationScript(LibraryInputScript):
     @classmethod
     def arg_parser(cls, _db):
-        parser = super(PatronInformationScript, cls).arg_parser(_db)
+        parser = super(PatronInformationScript, cls).arg_parser(_db, multiple_libraries=False)
```

Comment:
The help for this said that it could take multiple libraries before, which it actually can't. This change makes it show the correct help text. Probably good to change to make as well, while I'm here.